add-domain

```diff
@@ -522,3 +522,10 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
```

Comment:
Note for others.

This domain seems inactive at the writing moment; However the domain is still active, which means the DNS/web-server etc can be activated within milliseconds or only be response to specific regions / IP addresses, Hence leaving this record as ACTIVE